arquillian/appclient/pom.xml

```diff
@@ -0,0 +1,65 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+
+  <!-- Parent -->
+  <parent>
+    <groupId>jakarta.tck</groupId>
+    <artifactId>tck.arquillian.parent</artifactId>
```

Comment:
Should we create a pr with the jakarta.tck:tck.arquillian.parent:1.0.0-SNAPSHOT repo as well?  I don't seem to have this locally myself, is it new?

Reply:
Ahh, just found it https://github.com/arquillian/arquillian-tck